dependency-check-suppressions.xml

```diff
@@ -151,4 +151,14 @@
         <gav regex="true">.*</gav>
         <cve>CVE-2019-16869</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+           no fix available, not affected anyway
+           ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson-databind@.*$</packageUrl>
+        <cve>CVE-2019-12814</cve>
+        <cve>CVE-2019-14379</cve>
+        <cve>CVE-2019-14439</cve>
+        <cve>CVE-2019-12384</cve>
```

Comment:
All of the above exclusions were solved in master branch. I don't think you need to suppress them anymore. please remove these

Reply:
I can't see them in master

Reply:
because these are removed now. you can remove from your branch as well.

Reply:
done